core/trino-main/src/main/java/io/trino/execution/QueryManagerConfig.java

```diff
@@ -62,7 +62,7 @@
     private Duration remoteTaskMaxErrorDuration = new Duration(5, TimeUnit.MINUTES);
     private int remoteTaskMaxCallbackThreads = 1000;
 
-    private String queryExecutionPolicy = "all-at-once";
+    private String queryExecutionPolicy = "sequential";
```

Comment:
Not sure we want to switch the default. You mentioned in the commit message that overally `sequential` provides perf improvement. But are there queries which degraded with the change?

Reply:
> Not sure we want to switch the default. You mentioned in the commit message that overally sequential provides perf improvement. But are there queries which degraded with the change?

I don't think there are. Some small things:
* there might be slight CPU increase for some queries due to more concurrency now at source stage. Source stage
  concurrency is another topic to look at.
* `isAnyTaskBlocked` is not propagated by `Future` yet, but it shouldn't happen in practice too much because of 
  broadcast join limits).

Overall this is much, much better than current scheduler in high concurrency workloads and complex queries.
Hence IMO benefits greatly exceeds risks, which are small here.

Reply:
I guess low latency queries can degrade. For example if the build side is extremely small it might be better to start scheduling probe tasks right away.